docker/metaworker/Dockerfile

```diff
@@ -32,6 +32,11 @@ RUN apt-get update && \
         libmysqlclient-dev \
         libjpeg-dev \
         libpq-dev \
+        # selenium is a java thing
+        default-jre \
+        # chromium needs xvfb even with --headless
+        xvfb \
+        chromium-browser \
```

Comment:
@tardyp  maybe you also need `chromium-chromedriver`?

Last time I tried selenium tests at least on Ubuntu driver was in the separate package from `chromium-browser`: `chromium-chromedriver`.